packages/server-wallet/src/message-service/test-message-service.ts

```diff
@@ -28,10 +27,9 @@ export type LatencyOptions = {
  * All the engines will share the same message service.
  * The message service is responsible for calling pushMessage on the appropriate engines.
  */
-export class TestMessageService
-  extends EventEmitter<{deliveryRequested: {messages: Message[]}}>
-  implements MessageServiceInterface {
+export class TestMessageService implements MessageServiceInterface {
   private _handleMessages: (messages: Message[]) => Promise<void>;
+  private _sendMessage: ((messages: Message[]) => Promise<void>) | undefined = undefined;
```

Comment:
Am I right to understand that this change has some meaning? What is the meaning of this? Does it prevent some flickering test?

I recall some comment made in standup that sounds along the lines of what's changed here. I find it hard to understand what's going on in this context. Can you add some context to the commit message?

Reply:
I think this [comment](https://github.com/statechannels/statechannels/pull/3541#commitcomment-51395481) explains the reasoning behind the change. Let me know if that doesn't provide enough details!